### PR TITLE
3.18.0-preview: Adds changelog and contract files

### PR DESCRIPTION
**Pull request by ealsur**
# Pull Request Template

## Description

This bumps the preview SDK version to 3.18.0-preview.

## Preview Contract changes

image

image

image

### CODE REVIEW

**Comment by deparash** [comment]
Please wait for https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2308

File: Directory.Build.props [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<ClientOfficialVersion>3.17.0</ClientOfficialVersion>
-		<ClientPreviewVersion>3.17.0</ClientPreviewVersion>
-		<ClientPreviewSuffixVersion>preview1</ClientPreviewSuffixVersion>
+		<ClientOfficialVersion>3.17.1</ClientOfficialVersion>
```
Comment (deparash) [inline Directory.Build.props:4]: Please wait for https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2308

**Review by kirankumarkolli — DISMISSED** [review]

**Comment by ealsur** [comment]
@deparash added

File: changelog.md [hunk 2]
```diff
@@ -3,6 +3,19 @@ Preview features are treated as a separate branch and will not be included in th
...
+#### Added
+- [#2308](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2308) Patch: Adds preview support for Patch API
+- [#2312](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2312) Diagnostics: Adds Api for getting all regions contacted by a request
+
+#### Fixed
+- [#2311](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2311) Spatial: Fixes deserialization when Json does not represent a Spatial type
+- [#2284](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2284) Diagnostics: Adds traces for cache operations
+- [#2278](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2278) Documentation: Fixed typos in comment examples (Thanks to paulomorgado)
+- [#2279](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2279) Availability: Fixed region failover logic on control plane hot path when gateway hangs
+- [#2286](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2286) Diagnostics: Fixed regression which caused ActivityId to not get included
```
Comment (j82w) [inline changelog.md:17]: Diagnostics: Fixed regression which caused ActivityId to not get included. Introduced in 3.17.0 PR [#2097](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2097)

File: changelog.md [hunk 3]
```diff
@@ -3,6 +3,19 @@ Preview features are treated as a separate branch and will not be included in th
...
 
+### <a name="3.18.0-preview"/> [3.18.0-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.18.0-preview) - 2021-03-17
+
+#### Added
+- [#2308](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2308) Patch: Adds preview support for Patch API
+- [#2312](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2312) Diagnostics: Adds Api for getting all regions contacted by a request
+
+#### Fixed
+- [#2311](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2311) Spatial: Fixes deserialization when Json does not represent a Spatial type
+- [#2284](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2284) Diagnostics: Adds traces for cache operations
```
Comment (j82w) [inline changelog.md:14]: Diagnostics: Adds traces for cache operations. Introduced in 3.17.0 PR [#2097](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2097)

File: changelog.md (hunk 2 above)
Comment (ealsur) [reply changelog.md:17]: Done

**Review by j82w — APPROVED** [review]

**Review by j82w — APPROVED** [review]

**Pull request merged**